src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -219,7 +219,7 @@ private function getContainerDefinitionData(Definition $definition, bool $omitTa
                 if ($factory[0] instanceof Reference) {
                     $data['factory_service'] = (string) $factory[0];
                 } elseif ($factory[0] instanceof Definition) {
-                    $data['factory_class'] = $factory[0]->getClass() ?? 'not configured';
+                    $data['factory_class'] = sprintf('inline factory service (%s)', $factory[0]->getClass() ?? 'class not configured');
```

Comment:
should be `factory_service` being set here, not `factory_class`

Reply:
updated